source/app/pages/documents/index.js

```diff
@@ -226,6 +229,17 @@ function Documents({
               />
             : null }
 
+            {selectedSearch === 'both' &&
+              <TooltipButton
+                tooltip={<>
+                  <p>In a traditional keyword search, the results are privided in a list. The user needs to go through the list, select a document that may have the answer, and then go find the answer within the document.</p>
```

Comment:
Seems to be a typo here for *provided*

Reply:
Oh good spot, totally missed that - just pushed a fix for this